css/_jobcentre_theme.less

```diff
@@ -1255,6 +1255,20 @@ a.user-signup-link { color: #b3d4fc; }
     top: 400px;
     width: 84%;
 }
+.candidate-card {
+    display: inline-block;
+    width: 19.5%;
+    @media (max-width: 767px) {
+        width: 49%;
+    }
+    .tab-select {
+        width: 100%;
+        min-height: 58px;
+        @media (max-width: 767px) {
+            width: 100%;
+        }
+    }
+}
```

Comment:
I think this should be in _common.less, as cpajobs would benefit from this as well.

Reply:
Why is this `.candidate-card` in both `_common.less` and `_jobcentre_theme.less`? Isn't it redundant in `_jobcentre_theme.less`?